quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/services/QuarkusCompletions.java

```diff
@@ -56,7 +57,13 @@
 
 	private static final Logger LOGGER = Logger.getLogger(QuarkusCompletions.class.getName());
 
-	private static final List<String> DEFAULT_PROFILES = Arrays.asList("dev", "prod", "test");
+	private static final String DEV_PROFILE = "dev";

```

Comment:
To avoid having a lot of fields for each profile, I will work with EnumItem class (like BOOLEAN_ENUMS)

```java
private static final List<EnumItem> DEFAULT_PROFILES = Collections
			.unmodifiableCollection(Arrays.asList(new EnumItem("dev", "Activated when...")...
```

Reply:
Arrays.asList(...) creates immutable lists already